React-Native/TodoList/src/app/screens/Main/components/Input/styles.js

```diff
@@ -0,0 +1,10 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  input: {
+    padding: 15,
```

Comment:
you are already using this value `15` in another file.. so you could make a constant file like `dimensions.js` for all repeated constans for size